js/src/admin/components/ReadmeModal.js

```diff
@@ -0,0 +1,57 @@
+import app from '../../admin/app';
+import Modal from '../../common/components/Modal';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+import Placeholder from '../../common/components/Placeholder';
+
+export default class ReadmeModal extends Modal {
+  oninit(vnode) {
+    super.oninit(vnode);
+
+    this.name = this.attrs.extension.id;
+    this.displayName = this.attrs.extension.extra['flarum-extension'].title;
+
+    this.loading = true;
+
+    this.loadReadme();
+  }
+
+  className() {
+    return 'ReadmeModal Modal--large';
+  }
+
+  title() {
+    const name = this.displayName;
+    return app.translator.trans('core.admin.extension.readme.title', {
+      displayName: name,
```

Comment:
Could we do away with the extra `name` variable here and directly use `this.displayName()`?